README.md

```diff
@@ -1,23 +1,56 @@
-# Radius - A Design System Accelerator
+# Radius Workspace - A Design System Boilerplate Generator
 
-The Radius design system starter kit is a collection of open-source tools and libraries that guide and help you to build your design system faster. Design systems need to match your brand and product needs, and your current and future digital workflows. Too much, too little, too complex, or too off-brand, and you will fight your design system as much as benefit from it.
+## Background
+
+[rangle.io/radius](https://rangle.io/radius)
+[Radius Design Kit](https://www.figma.com/file/RqENxZWAzGiEWM7COch1Sc/Radius-Design-Kit) on Figma
+
+The Radius design system starter kit is a collection of open-source tools and libraries developed by Rangle to guide devs to build design systems faster. Design systems need to match increasingly flexible brand and product needs, to adapt to everchanging digital workflows.
+
+`radius-workspace` is 1 of 3 repositories that constitute the Radius ecosystem for accelerating the building of design systems!
+
+This workspace reflects our internal R&D efforts to:
+
+- automate the generation of design system starter boilerplates
+- offer the choice of popular choices around tooling, JS frameworks and style libraries
+- enable the easy translation of Figma design tokens into styles within your new boilerplate that your dev team can readily work with and scale
+
+See also:
+[Radius-Angular](https://github.com/rangle/radius-angular)
+[Radius](https://github.com/rangle/radius) [for React]
+
+## Create Design System instances with the Radius CLI
 
-The Radius is an opinionated assemblage of open-source tools and libraries that allow you to accelerate your design system. The choice of tools, their composition, and a set of foundational components guides you on how to build a constraint-based system.
+1. Instantiate a design system boilerplate!
+   `npx radius-script init`
```

Comment:
we don't want to mention `radius-script` that is just for testing.

Reply:
I'm thinking about what a dev without the global install can do, as I just heard from Rafae and Doga that you can run everything related to the cli with npm / yarn. Is that an attractive option?